### PR TITLE
VIH-7323 participants list improvements

### PR DESCRIPTION
**Pull request by phillipjchambers**
### JIRA link (if applicable) ###

VIH-7323

### Change description ###

* apply style changes as per VIH-7323
* split participant list across two components - list & item
* refactor both components
* update tests

OLD
image

NEW
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by RuwanikaWeerakkody — DISMISSED** [review]

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<div *ngIf="participant" class="govuk-grid-row container">
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.html:1]: I really like this style!

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts [hunk 2]
```diff
@@ -19,11 +19,11 @@ import { Logger } from '../../services/logger';
 import { SearchService } from '../../services/search.service';
 import { VideoHearingsService } from '../../services/video-hearings.service';
 import { RemovePopupStubComponent } from '../../testing/stubs/remove-popup-stub';
-import { ParticipantsListComponent } from '../participants-list/participants-list.component';
 import { SearchEmailComponent } from '../search-email/search-email.component';
 import { ParticipantService } from '../services/participant.service';
 import { AddParticipantComponent } from './add-participant.component';
 import { HearingRoleModel } from '../../common/model/hearing-role.model';
+import { ParticipantListComponent } from '../participant';
```
Comment (levibutler) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:26]: other component file names end in .component
should this be called `participant-list` (`-component`) as its not just a participant?

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/index.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export { ParticipantListComponent } from './list/participant-list.component';
```
Comment (levibutler) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/index.ts:1]: file name and or directory correct?

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
+:host {
+  display: block;
+  border-color: #cccccc;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 11px;
+  padding: 15px;
+
+  // these shouldn't be here really
```
Comment (levibutler) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.scss:9]: a forgotten todo?

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.spec.ts [hunk 5]
```diff
@@ -22,7 +22,7 @@ import { VideoHearingsService } from '../../services/video-hearings.service';
 import { MockValues } from '../../testing/data/test-objects';
 import { BookingEditStubComponent } from '../../testing/stubs/booking-edit-stub';
 import { ParticipantsListStubComponent } from '../../testing/stubs/participant-list-stub';
-import { ParticipantsListComponent } from '../participants-list/participants-list.component';
+import { ParticipantListComponent } from '../participant/list/participant-list.component';
```
Comment (levibutler) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.spec.ts:25]: was this the name of the file i commented on earlier?

**Review by shaed-parkar — DISMISSED** [review]

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts (hunk 2 above)
Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:26]: `participant` is just the folder - I've added an index.ts to shorten the paths a bit

Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:26]: that folder contains both the `list` and the `item`

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.spec.ts (hunk 5 above)
Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.spec.ts:25]: You're right - it is. I can shorten that import.

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.scss (hunk 4 above)
Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.scss:9]: Removed!

[the remark this answers is not in the document]
Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.scss:9]: project is set up wrong - I'm just using the same as was there before. If you remove these, you get Times.

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.ts [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    styleUrls: ['./participant-list.component.scss']
+})
+export class ParticipantListComponent implements OnInit {
+    private readonly loggerPrefix = '[ParticipantList] -';
+    @Input()
+    participants: ParticipantModel[] = [];
+
+    sortedParticipants: ParticipantModel[] = [];
+
+    $selectedForEdit = new EventEmitter<string>();
```
[the remark this answers is not in the document]
Comment (phillipjchambers) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/list/participant-list.component.ts:18]: Yep - just had to draw a line in the refactoring somewhere...

**Review by levibutler — APPROVED** [review]
LGTM 👍

**Pull request merged**